docker-compose.yml

```diff
@@ -1,14 +1,23 @@
-version: '3.3'
+version: '3.8'
 
 services:
-    mongo:
-        image: mongo
+  mongo:
+    image: mongo:4
+    restart: always
+    networks:
+      backend:
 
-    mongoexpress:
-        image: mongo-express
-        ports:
-            - "8081:8081"
-        links:
-            - mongo
-        environment: 
-            - ME_CONFIG_MONGODB_URL=mongodb://mongo:27017
\ No newline at end of file
+  mongoexpress:
+    image: mongo-express
+    ports:
+      - "8081:8081"
+    environment: 
+      - ME_CONFIG_MONGODB_URL=mongodb://mongo:27017
+    depends_on:
+      - mongo
+    restart: always
+    networks:
+      backend:
+
+networks:
+  backend:
```

Comment:
Can we drop the custom network? By default, compose files will create a network for all components called something like "project_default". If I'm not mistaken, if you already had a network called "backend" this will join it, that doesn't seem ideal.

Reply:
Done, removed network